bootstrap/AWS_SETUP.md

```diff
@@ -0,0 +1,199 @@
+# Preparing Amazon Web Services for Github Foundations
+
+This document will walk you through what is required of your Amazon Web Services (AWS) setup to run the Github Foundations bootstrap layer.
+
+## Setup
+
+**1. Create an IAM Policy**
+* Create an IAM Policy on the AWS account that the terraform state will be stored in. The policy requires the following minimum permissions:
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "iam:ListRoleTags",
```

Comment:
Would it be possible to group these by resource, example just having all the IAM., DynamoDB, S3 etc in one group? That way it is easier to see what permissions are required. Thank you!